Week2/assign-string.js

```diff
@@ -0,0 +1,7 @@
+console.log("string is zero or more characters written inside quotes.");
+var y = "huseyin";
+console.log(y);
```

Comment:
Ah, here again, your explanation is correct, but waht do you think the value of `y` is?